src/homeassistant.cpp

```diff
@@ -569,3 +572,73 @@ void HomeAssistant::onHeartbeatTimeout() {
         },
         param);
 }
+
+QStringList HomeAssistant::supportedFeatures(const QString &entityType, const int &supportedFeatures) {
```

Comment:
Are the "feature strings" used for all integrations and part of the API? (Sorry I haven't looked into it yet).
Re-defining the same string constants all over the place is bad practice, very error prone and hard to change.
They should be defined once as constants in a header file.

Reply:
This method is matching the features used by Home Assistant with the features YIO uses.